### PR TITLE
release 1.3.4

### PR DESCRIPTION
**Pull request by ndesai-newrelic**

### CODE REVIEW

File: android/src/main/java/com/newrelic/capacitor/plugin/NewRelicCapacitorPluginPlugin.java [hunk 1]
```diff
@@ -384,13 +414,49 @@ public void noticeHttpTransaction(PluginCall call) {
             return;
         }
 
-        JSONObject traceAttributes = call.getObject("traceAttributes");
+                JSONObject traceAttributes = call.getObject("traceAttributes");
         Map<String, Object> traceHeadersMap = new HashMap<String, Object>();
```
Comment (ywang-nr) [inline android/src/main/java/com/newrelic/capacitor/plugin/NewRelicCapacitorPluginPlugin.java:418]: space here?

**Review by ywang-nr — APPROVED** [review]
Have two minor questions

File: README.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ AppToken is platform-specific. You need to generate separate tokens for Android
       }
       dependencies {
         ...
-        classpath "com.newrelic.agent.android:agent-gradle-plugin:7.0.0"
+        classpath "com.newrelic.agent.android:agent-gradle-plugin:7.2.0"
```
[the remark this answers is not in the document]
Comment (ndesai-newrelic) [reply README.md:111]: i can do that.

File: android/src/main/java/com/newrelic/capacitor/plugin/NewRelicCapacitorPluginPlugin.java (hunk 1 above)
Comment (ndesai-newrelic) [reply android/src/main/java/com/newrelic/capacitor/plugin/NewRelicCapacitorPluginPlugin.java:418]: let me fix this.

**Comment by ywang-nr** [comment]
Some of the unit tests failed, would that be okay?

**Comment by ndesai-newrelic** [comment]
@ywang-nr let me fix those issues.

**Review by ywang-nr — APPROVED** [review]

**Pull request merged**